posts/2024-02-15-waku-network-tech-overview.mdx

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+name: 'The Waku Network: Technical Overview'
+title: 'The Waku Network: Technical Overview'
+date: 2024-02-15 00:00:00
+authors: [franck]
+published: true
+slug: 2024-waku-network-tech-overview
+categories: waku, technical, network
+image: /img/black-waku-logo-with-name.png
+hide_table_of_contents: false
+---
+
+# The Waku Network - Technical Overview and Properties
+
+<!--
+Goals:
+
+- Introduce the Waku Network: Why, what how
+- Introduce RLN
+- State of things
+- USPs
+- Work moving forward
+-->
+
+Last December [we announced The Waku Network](https://thedefiant.io/waku-launches-first-decentralised-privacy-preserving-dos-protections-for-peer-to-peer-messaging).
+
+In the press release, we defined the Waku Network as the first decentralized and privacy-preserving network that provides Denial-of-Service (DoS) protections for peer-to-peer messaging. Aiming to enhance privacy and security in peer-to-peer communication by implementing innovative protocols and technologies.
+
+Let’s dig a bit deeper on the why and what is The Waku Network.
```

Comment:
'Let us'

We should stick with not using contractions throughout the article

Reply:
We should also standardise capitalisation of 'the Waku Network'

Reply:
@Amelia7689 are you able to take over and proceed with corrections? feel free to open a 2nd PR and I can close this one.

Here is a link to get the file: https://github.com/waku-org/blog.waku.org/blob/waku-network-tech/posts/2024-02-15-waku-network-tech-overview.mdx